From 4307d5d0ad3d6632bafbc92ab612265c06d9fc3d Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 27 Nov 2023 21:09:40 -0800
Subject: [PATCH] Update configparser method

---
 versioneer.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/versioneer.py b/versioneer.py
index 64fea1c..89c032a 100644
--- a/versioneer.py
+++ b/versioneer.py
@@ -339,9 +339,9 @@ def get_config_from_root(root):
     # configparser.NoOptionError (if it lacks "VCS="). See the docstring at
     # the top of versioneer.py for instructions on writing your setup.cfg .
     setup_cfg = os.path.join(root, "setup.cfg")
-    parser = configparser.SafeConfigParser()
+    parser = configparser.RawConfigParser()
     with open(setup_cfg, "r") as f:
-        parser.readfp(f)
+        parser.read_file(f)
     VCS = parser.get("versioneer", "VCS")  # mandatory
 
     def get(parser, name):
-- 
2.43.0

